docs/configuration.md

```diff
@@ -113,6 +113,7 @@ Unless you need to set a non-default value, it is recommended to only populate o
 | `WS_CONNECTION_LIMIT_PER_IP`    | "10"     | Maximum amount of connections from a single IP address                                                                        |
 | `WS_MULTIPLE_ADDRESSES_ENABLED` | "false"  | If enabled eth_subscribe will allow subscription to multiple contract address.                                                |
 | `WS_CACHE_TTL`                  | "20000"  | The time to live for cached entries.                                                                                          |
+| `WS_NEW_HEADS_ENABLED`.         | "true".  | Enables subscriptions for the lates blocks, `newHeads`.                                                                       |
```

Comment:
Default should be false until we've tested

Reply:
Done.